src/pages/About.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+
+export const About = () => {
+  return (
+  <div>
+    <h2>About This App</h2>
+    <h5>App to search Github users<br/>
+    Version 1.0.0
+    </h5>
+  </div>)
+}
```

Comment:
- Format this file properly
- Fix all eslint error including the `arrow-body-style` error.

Reply:
okay done